scripts/travis/build.sh

```diff
@@ -62,6 +64,8 @@ case "$JOB" in
     #   - or the branch distTag from package.json is "latest" (i.e. stable branch)
     if [[ "$TRAVIS_TAG" != '' || "$TRAVIS_BRANCH" == master || "$DIST_TAG" == latest ]]; then
       DEPLOY_CODE=true
+    else
+      DEPLOY_CODE=false
     fi
 
     if [[ "$DEPLOY_DOCS" || "$DEPLOY_CODE" ]]; then
```

Comment:
I think this will always evaluate to true, even if both are `false` (because `false` is truthy :smiley:)

Reply:
That's right, good catch. Bash script doesn't have boolean true / false.
You can see here how this condition fails, but the actual deploy conditions are correct: https://travis-ci.org/angular/angular.js/jobs/290626171